service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageQueries.kt

```diff
@@ -63,12 +64,11 @@ fun Database.Transaction.insertMessage(
     sender: MessageAccountId,
     recipientNames: List<String>,
     repliesToMessageId: MessageId? = null,
-    sentAt: HelsinkiDateTime? = HelsinkiDateTime.now()
 ): MessageId {
     // language=SQL
     val insertMessageSql = """
         INSERT INTO message (content_id, thread_id, sender_id, sender_name, replies_to, sent_at, recipient_names)
-        SELECT :contentId, :threadId, :senderId, name_view.account_name, :repliesToId, :sentAt, :recipientNames
+        SELECT :contentId, :threadId, :senderId, name_view.account_name, :repliesToId, now(), :recipientNames
```

Comment:
probably doesn't matter here but in general this decreases testability

Reply:
True. I had originally implemented the `sentAt` as a parameter because I was not sure about how the default value within a transaction worked.